Makefile

```diff
@@ -105,6 +105,15 @@ $(PACKAGE_PATH)/resources/mvnw:
 		&& rm -r maven-wrapper-distribution-3.1.1-bin.zip \
 		&& echo -e "distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.6/apache-maven-3.8.6-bin.zip\nwrapperUrl=https://repo.maven.apache.org/maven2/org/apache/maven/wrapper/maven-wrapper/3.1.1/maven-wrapper-3.1.1.jar" > .mvn/wrapper/maven-wrapper.properties \
 		&& cd $(REPO_PATH)
+$(PACKAGE_PATH)/resources/gradlew:
+	cd $(PACKAGE_PATH)/resources \
+	&& export GRADLE_VERSION=7.6 \
```

Comment:
The indentation of this block is not consistent compare to `$(PACKAGE_PATH)/resources/mvnw`.